src/Horarium.Mongo/MongoRepository.cs

```diff
@@ -108,6 +109,19 @@ public async Task RemoveJob(string jobId)
             await collection.DeleteOneAsync(x => x.JobId == jobId);
         }
 
+        public async Task RescheduleRecurrentJob(string jobId, DateTime? startAt, Exception error)
+        {
+            var collection = _mongoClientProvider.GetCollection<JobMongoModel>();
+
+            var update = Builders<JobMongoModel>.Update
+                .Set(x => x.StartAt, startAt)
+                .Set(x => x.Status, JobStatus.Ready)
+                .Set(x => x.Error, error.Message + error.StackTrace);
```

Comment:
Possible Null reference exception in case of `error` is null at https://github.com/TinkoffCreditSystems/Horarium/pull/44/files#diff-8df2ae70f726bf17811347a4c63e5d417e66e4b05ca476711bc93a52b374f8f1R110